chain-time/src/timeframe.rs

```diff
@@ -6,7 +6,7 @@ use std::time::{Duration, SystemTime};
 /// The slots are not comparable to others slots made on a
 /// different time frame
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
-pub struct Slot(pub(crate) u64);
+pub struct Slot(pub u64);
```

Comment:
does it need to be public? I think this value, the only way we should get it, should be through the `TimeFrame` object. It's a bit annoying but this is the only way to guarantee this object is not constructed with the wrong value, or with value that does not make sense in a given timeframe.

Reply:
I've fixed the test to use `ledger.settings.era` so it doesn't need to construct `Slot` anymore.